node-launchpad/src/widgets/hyperlink.rs

```diff
@@ -26,6 +27,20 @@ impl<'content> Hyperlink<'content> {
     }
 }
 
+// Displays the hyperlink in the terminal using OSC 8.
+// Underline solid \x1b[4m
+// Foreground color 45 \x1b[38;5;45m
+impl fmt::Display for Hyperlink<'_> {
```

Comment:
For a follow up PR, we should reomve the `HyperLink` struct or try to modify it to fit the use cases.